### ISSUE TITLE
Azerty keyboard problem

### ISSUE THREAD

**Issue by BriceTouchard** [hetuw/OneLife#24]
I have a french azerty keyboard, so I tried to change the W and A key for respectively Z and Q, to be able to use "WASD" controls (ZQSD in my case).
The problem is when you hit Z to go up or Q to go left, it is like the key is still pressed (even though it's not) so the character keep going in the same direction. To stop the character from moving you have to press W or A ... weird ... 

Maybe the configuration file only set the _press_ key and not the _release_ key ?

So for now I changed the keyboard configuration on windows without changing the .cfg file, but it's annoying.

Amazing mod btw and thanks in advance.

**Comment by PXshadow** [comment]
Yeah I just talked to another person with a french keyboard they have the same issue, Internally OneLife uses SDL and the layer above has a strange hard code setup for WASD keyboard using ASCII so I don't think it will work as expected because of that. You might be able to set totally different keys like Q to go up but I don't have a french keyboard to test. I'd like to fix this problem but it's quite a hard process to go through this type of code base and fix. Perhaps another person that has another set keyboard could make a universal key char model.

**Comment by Abtacha** [comment]
I am using NEO2 layout and have the same issue. I don't have much coding experience but I will have a look into this and at least try.
